### DISCUSSION TITLE
Host meli-email.org not found: 2(SERVFAIL)

### DISCUSSION THREAD

**Discussion by stappersg** [meli/meli#27] [Q&A] [answered]
Over here
```text
stappers@rosa:~
$ host meli-email.org
Host meli-email.org not found: 2(SERVFAIL)
stappers@rosa:~
$ 
```
And from another host in Europe:
```text
stappers@hop:~
$ host meli-email.org
Host meli-email.org not found: 2(SERVFAIL)
stappers@hop:~
$ 
```

Who cares about that issue?

**Comment by epilys** [comment]
Hi @stappersg, I contacted our nameserver authority as soon as I got an alarm yesterday. For some reason all our .org domains resolve except for meli-email.org. I will try temporarily converting the domain alias meli-mua.org to the primary one instead of redirecting to meli-email.org.

**Comment by epilys** [comment]
Main repo should be accessible here, https://git.meli-mua.org/meli/meli 

Once the main url is functional this link will start redirecting to the proper domain.

**Comment by stappersg** [comment, marked as answer]
And https://meli-email.org/ is again available

![image](https://github.com/meli/meli/assets/35434094/d767f3b4-25df-46ce-8a0c-c8f8a5d2f2cb)
